src/components/resize-listener-mixin.cjsx

```diff
@@ -0,0 +1,67 @@
+React = require 'react'
+_ = require 'underscore'
+
+module.exports =
+  propTypes:
+    resizeThrottle: React.PropTypes.number
+
+  getDefaultProps: ->
+    resizeThrottle: 200
+
+  getInitialState: ->
+    $window: {}
```

Comment:
What is the reason for the distinction between `$` and `_` in the state? I'm guessing you have a prefix because this is a mixin?